client/tests/webdriver/pages/editPosition.page.js

```diff
@@ -23,6 +24,12 @@ class EditPosition extends Page {
         }
     }
 
+    waitForOrgAutoCompleteToChange() {
```

Comment:
I think we should give the name of the org as a parameter to the method. This way we could reuse the method also for principal org and also for no suggestion found.

Reply:
I also think you could also add here the wait for exists and wait for visible.